src/GraphMap/Node.cs

```diff
@@ -0,0 +1,21 @@
+public class Node

```

Comment:
Maybe we should also consider renaming this class to something like `Location`?

Reply:
Indeed, I also renamed 'Edge' to 'Trail'